.github/workflows/ci.yml

```diff
@@ -1,30 +1,131 @@
----
-
 name: CI
 
 on:
   pull_request:
-    branches: [devel]
+    branches:
+      - devel
 
   push:
-    branches: [devel]
+    branches:
+      - devel
 
 jobs:
+  sanity:
+    name: ${{ matrix.test.name }}
+    runs-on: ubuntu-20.04
+    container:
+      image: quay.io/ansible/ansible-builder-test-container:1.0.0
+      env:
+        PIP_CACHE_DIR: ${{ runner.temp }}/.cache/pip
+
+    strategy:
+      fail-fast: false
+      matrix:
+        test:
+          - name: Lint
+            env: linters
+
+          - name: Docs
+            env: docs
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup test environment
+        run: tox -e ${{ matrix.test.env }} --notest
+
+      - name: Run tests
+        run: tox -e ${{ matrix.test.env }}
+
+      - name: Save docs
```

Comment:
You probably don't need this if you just enable RTD. It'll have PR previews out of the box, with no expiration.

Reply:
I was trying to duplicate the functionality in Zuul that previews docs builds for PRs. I didn't know that was something that Read the Docs could do.

Reply:
Here's an example of how it looks in the checks section: https://github.com/ansible/pylibssh/pull/250.